packages/text-bitmap/src/utils/drawGlyph.ts

```diff
@@ -47,25 +47,17 @@ export function drawGlyph(
     context.fillStyle = generateFillStyle(canvas, context, style, resolution, [char], metrics);
     context.strokeStyle = style.stroke as string;
 
-    context.font = style.toFontString();
-    context.lineWidth = style.strokeThickness;
-    context.textBaseline = style.textBaseline;
-    context.lineJoin = style.lineJoin;
-    context.miterLimit = style.miterLimit;
-
-    // set canvas text styles
-    context.fillStyle = generateFillStyle(canvas, context, style, resolution, [char], metrics);
-    context.strokeStyle = style.stroke as string;
```

Comment:
Are you sure these were meant to be deleted? This seems inconsistent with the original PR

Reply:
revert back that code block. Code looked like duplicate.

Reply:
Oh sorry @goldenratio my mistake, you were also cherry-picking #7489 as well. You can remove those lines.

Reply:
removed